crane/cmd.go

```diff
@@ -192,6 +193,21 @@ formatted in bold provided the 'colorize' flag is on.`,
 		}, true),
 	}
 
+	var cmdTmpl = &cobra.Command{
+		Use:   "template",
+		Short: "Generates a textual output using Go templates",
+		Long: `Generates a textual output using Go templates. Use the 'template' flag to
+indicate the input template. Depending on the optional 'output' flag's value the
+behaviour is different. If you leave it, the output will be dumped through Stdout.
+If you specify a filename, the output will be rendered on that file. Additionally,
```

Comment:
`Additionally` makes me think that `the output will be rendered on that file` remains true in the `%s` case, which is not the case, it really is an either-or.